packages/kbn-apm-synthtrace/src/cli/utils/bootstrap.ts

```diff
@@ -47,15 +48,23 @@ export async function bootstrap(runOptions: RunOptions) {
     concurrency: runOptions.concurrency,
   });
 
+  const infraEsClient = getInfraEsClient({
+    target: esUrl,
+    logger,
+    concurrency: runOptions.concurrency,
+  });
+
   if (runOptions.clean) {
     await apmEsClient.clean();
     await logsEsClient.clean();
+    await infraEsClient.clean();
```

Comment:
@dgieselaar do you think it's fine to run these promises in parallel?
```suggestion
    await Promise.all([apmEsClient.clean(), logsEsClient.clean(), infraEsClient.clean()])
```

Reply:
@cauemarcondes What is your concern here? They are different clients using different indices and when I tested it they were all cleaned without any errors. Maybe I am missing something 🤔 
@achyutjhunjhunwala I saw you added the logs client - were there any issues with having the promises in parallel there?
cc: @dgieselaar

Reply:
I think it's fine for now, it only deletes data streams.